url-updater/src/main/java/com/devonfw/tools/ide/url/tool/aws/AwsUrlUpdater.java

```diff
@@ -27,6 +29,13 @@ protected String getGithubRepository() {
     return "aws-cli";
   }
 
+  /**
+   * @return the base url.
+   */
+  protected String getBaseUrl() {
```

Comment:
IMHO this method is overridden...

```suggestion
  @Override
  protected String getBaseUrl() {
```

Reply:
I've introduced an abstract getBaseUrl method and made sure that all base urls for the GithubUrlUpdater can be overwritten in tests now.